utils/spdx/src/main/kotlin/model/SpdxExternalDocumentReference.kt

```diff
@@ -48,6 +48,10 @@ data class SpdxExternalDocumentReference(
             "The external document ID must start with '${SpdxConstants.DOCUMENT_REF_PREFIX}'."
         }
 
-        require(spdxDocument.isNotBlank()) { "The SPDX document must not be blank." }
+        require(spdxDocument.isNotEmpty()) { "The SPDX document must not be empty." }
+
+        require(spdxDocument.trim() == spdxDocument) {
+            "The SPDX document must not contain any leading or trailing whitespace."
```

Comment:
Ideally, this change should be accompanied by a test case.

Reply:
We have a lot of such assertions in almost all SPDX model classes, and none of them is covered by explicit tests so far. So I don't feel like adding a test for this specific assertion right now, TBH.

Reply:
Somewhere we have to start with improving test coverage. But I won't insist.